spec/features/assignment_creation_spec.rb

```diff
@@ -0,0 +1,645 @@
+require 'rails_helper'
+require 'spec_helper'
+
+def GenerateAssignmentName()
+	(rand(1000) + 1).to_s + 'RSpecID' + (1 + rand(1000)).to_s
+end
+
+RSpec.feature "create private assignment"  do
+    before(:each) do |example|
+      unless example.metadata[:skip_before]
+         visit root_path
+         fill_in('login_name', :with => 'instructor6')
+         fill_in('login_password', :with => 'password')
+         click_on('SIGN IN')
+         expect(page).to have_content('Manage',:wait=>2)
+         within(".content") do
+         click_on("Assignments")
+         click_button 'New private assignment'
+         fill_in('assignment_form_assignment_name',:with => GenerateAssignmentName())
+         select('CSC 517 Fall 2010', from: 'assignment_form_assignment_course_id')
```

Comment:
This is test is not going to pass Travis CI because in the test db there will not be such course. In travic CI, there is only data defined in seeds.rb. It is not a good idea to suppose the full production db is available for testing. However, it is okay that you create some courses,  assignments, or users for testing use.